File: .NET/Microsoft.Recognizers.Text.Sequence/Models/HashTagModel.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.Recognizers.Text.Sequence
{
    public class HashtagModel : AbstractSequenceModel
    {
        public HashtagModel(IParser parser, IExtractor extractor)
            : base(parser, extractor)
        {
        }

        public override string ModelTypeName => Constants.MODEL_HASHTAG;
    }
}

File: Java/libraries/recognizers-text-sequence/src/main/java/com/microsoft/recognizers/text/sequence/models/HashTagModel.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.recognizers.text.sequence.models;

import com.microsoft.recognizers.text.IExtractor;
import com.microsoft.recognizers.text.IParser;
import com.microsoft.recognizers.text.sequence.Constants;

public class HashTagModel extends AbstractSequenceModel {
    public HashTagModel(IParser parser, IExtractor extractor) {
        super(parser, extractor);
        this.modelTypeName = Constants.MODEL_HASHTAG;
    }
}
